src/portal/README.rst

```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'portal' Extension
+==========================================
+
+This package is for the 'portal' extension.
+i.e. 'az portal'
```

Comment:
Please put more content in this readme file introducing this extension. An example is [here](https://github.com/Azure/azure-cli-extensions/tree/master/src/storage-preview).